server/services/store/sqlstore/blocks.go

```diff
@@ -473,3 +524,38 @@ func (s *SQLStore) GetBlockCountsByType() (map[string]int64, error) {
 	}
 	return m, nil
 }
+
+func (s *SQLStore) GetBlock(c store.Container, blockID string) (*model.Block, error) {
+	query := s.getQueryBuilder().
+		Select(
+			"id",
+			"parent_id",
+			"root_id",
+			"created_by",
+			"modified_by",
+			s.escapeField("schema"),
+			"type",
+			"title",
+			"COALESCE(fields, '{}')",
+			"create_at",
+			"update_at",
+			"delete_at",
+		).
+		From(s.tablePrefix + "blocks").
+		Where(sq.Eq{"id": blockID}).
+		Where(sq.Eq{"coalesce(workspace_id, '0')": c.WorkspaceID})
+
+	rows, err := query.Query()
+	if err != nil {
+		s.logger.Error(`GetBlock ERROR`, mlog.Err(err))
+		return nil, err
+	}
+
+	blocks, err := s.blocksFromRows(rows)
```

Comment:
This error is being ignored.  I'm not sure if empty result set is considered an error, but either way that can be checked and if a true error then err should be returned.

Reply:
Yes, this was missed.